test/rules.jl

```diff
@@ -46,4 +50,8 @@ _second(t) = Base.tuple_type_head(Base.tuple_type_tail(t))
     @test nice_pushforward === 0
     rrx, nice_pullback = rrule(nice, 1)
     @test (NO_FIELDS, 0) === nice_pullback(1)
+
+    sx = @SVector [1, 2]
+    sy = @SVector [3, 4]
+    @inferred frule(very_nice, 1, 2, Zero(), sx, sy)
```

Comment:
Maybe a comment or a testset name to make it clear why we are doing this?
```suggestion
    # This actually is testing that @scalar_rule and `One()` play nice together, w.r.t broadcasting
    @inferred frule(very_nice, 1, 2, Zero(), sx, sy)
```